docker-compose.yml

```
@@ -1,8 +1,7 @@
-version: "3"
 services:
   pihole:
     container_name: pihole
-    image: pihole/pihole:2022.05
+    image: "pihole/pihole:${TAG}"
```

Ah this is what I was missing w/r/t the env var (aside from misreading it). I assumed TAG was a well known env var for docker compose. Can we add a [default](https://docs.docker.com/compose/environment-variables/variable-interpolation/#interpolation-syntax) here so it'll run ok without any extra vars? The `PIHOLE_*` ones are just so the TF client can connect, TAG would be the only one required for the server as is. This way we can move it as required from the docs. What do you think?

```suggestion
    image: "pihole/pihole:${TAG:-latest}"
```

---

Thought about it, I was already late to review this PR, I'll just merge this in and do a quick update on my end